en/admin/asynchronous.md

```diff
@@ -48,13 +46,11 @@ rabbitmq_prefetch_count: 10 # read http://www.rabbitmq.com/consumer-prefetch.htm
 
 ### Enable RabbitMQ in wallabag
 
-In internal settings, in the **Import** section, enable RabbitMQ (with
-the value 1).
+In internal settings, in the **Import** section, enable RabbitMQ (with the value 1).
 
 ### Launch RabbitMQ consumer
 
-Depending on which service you want to import from you need to enable
-one (or many if you want to support many) cron job:
+Depending on which service you want to import from you need to enable one (or many if you want to support many) cron job:
```

Comment:
Why are you removing 80-char line limits?

Reply:
it is completely inconsistent all over this repo